BrainPortal/lib/cbrain_task_generators/templates/bourreau.rb.erb

```diff
@@ -505,8 +505,8 @@ class CbrainTask::<%= name %> < <%= (descriptor['custom'] || {})['cbrain:inherit
         # Print a warning if the file is not present
         # Note that, since the output may be optional, the process may still be successful
         unless path.present? && File.exists?(path)
-          self.addlog("Unable to find optional output file: #{path}") if isOptional.(param)
-          succeeded = false if ! isOptional.(param)
+          self.addlog("Unable to find optional output file: #{path}") if isOptional.(param.to_sym)
```

Comment:
Instead of requiring the user of the lambda to invoke `to_sym` on the arguments each and every time (like you did here), why don't have the lambda invoke `to_sym` itself? The lambda is it's own domain, so it know what its internal types are (the array of symbols) and it know what transformations to do. You, as a user of hte lambda, should not have to know that.